packages/components/scss/BaseStyleSheet.scss

```diff
@@ -755,7 +757,7 @@ input[type='number']::-webkit-inner-spin-button {
       center;
     //make the dotted duplicate focus line on firefox go away
     &:-moz-focusring {
-      color: rgba(0, 0, 0, 0%);
+      color: transparent;
```

Comment:
Dropdown arrow in the select input is appearing black:
![image](https://github.com/deephaven/web-client-ui/assets/4505624/a22cf4b7-c756-4b77-b420-8633e80e090d)

Reply:
This is gonna be slightly annoying to fix. My best suggestion is we need to change all usages of `<select>` to use `<Select>` (22 in core, 35 in enterprise). Then wrap `Select` with an extra element to add an icon that we can colorize with a css variable. Probably want to do this in it's own PR first.

Reply:
This is due to a limitation with inline SVGs not supporting injected CSS variables. I put in a static color value to address this temporarily. @dsmmcken is planning to change the implementation of our <select> to make it possible to style dynamically.

Reply:
@bmingles and I discussed a different plan since this is also an issue for input invalid and select invalid markers.

New plan is to make css variables that hold the the icon, and then use JS to string interpolate a resolved color into the variable on the client when it is being switched.